bin/diagnostic.js

```diff
@@ -1 +1,13 @@
 'use strict'
+const ask = require('../lib/ask.js')
+
+let answer = 'not empty'
+
+while (answer !== 8) {
+  answer = ask('Guess my number')
+  if (answer >= 1 && answer <= 10) {
+    console.log('You guessed it!')
+  } else {
+    console.log('Guess needs to be between 1 and 10')
+  }
+}
```

Comment:
Great job Van! You were so close. Just to pretty it up I would add a `?` and a blank space to the end of the string `'Guess my number'` 

Then you also want to add a `break` statement after printing `'You guessed it!'` otherwise it will continue looping forever!